content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -66,6 +66,13 @@ of the form `auto-generated-name.my-svc.my-namespace.svc.cluster-domain.example`
 
 ## Pods
 
+### A/AAAA records
+
+Any pods created by a Deployment or DaemonSet have the following
+DNS resolution available:
+
+`pod-ip-address.deployment-name.my-namespace.svc.cluster-domain.example.`
```

Comment:
I could be wrong, but if you know the pod IP address what purpose does the DNS record serve?
```suggestion
`pod-name.deployment-name.my-namespace.svc.cluster-domain.example.`
```

Reply:
Couldn't agree more but I just documented the way how it currently works, changing it in docs will not fix it in general :) 

Would be great to have some insights from sig-network guys on existing implementation.

Reply:
Hey @ytsarev and @jimangel - I pinged SIG Network in Slack and we’ll see if someone is able to audit this PR 👍🏼